src/apps/schema/src/app/components/Field/index.tsx

```diff
@@ -282,7 +282,7 @@ export const Field = ({
             fontWeight="700"
             noWrap
           >
-            {field.label}
+            {field.label} {"required" in field && field.required && "*"}
```

Comment:
@shrunyan it's to satisfy typescript. Without it I had to do some typecasting which Andres previously commented on avoiding, without the "in" check, ts will throw an error since `field` can be different types and "required" may not be on those other types.